src/plugins/hidsim/gui/vmi_win_gui_utils.cpp

```diff
@@ -181,8 +181,12 @@ char* retrieve_objhdr_name(vmi_instance_t vmi, addr_t addr)
  */
 wchar_t* read_wchar_str_pid(vmi_instance_t vmi, addr_t start, size_t len, vmi_pid_t pid)
 {
-    wchar_t* s = (wchar_t*) malloc(sizeof(wchar_t) * len);
-    memset(s, 0, sizeof(wchar_t) * len);
+    wchar_t* s = (wchar_t*) calloc(1, sizeof(wchar_t) * len);
```

Comment:
I think the correct way of calloc usage here is:
wchar_t* s = (wchar_t*) calloc(len, sizeof(wchar_t));

This is better because calloc has internal check for size multiplication overflow (sizeof * len)

Reply:
Makes total sense, thanks @exescript! Fixed in 44490fb04916dc41686d1d714bde7758b5fd598d.